testing/kuttl/e2e/root-cert-ownership/03-check-owners.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  # Get a list of the current owners of the root ca cert secret and verify that
+  # owner1 is no longer listed and owner2 is found.
+  - script: |
+      sleep 2 # this sleep allows time for the owner reference list to be updated
+      CURRENT_OWNERS=$(kubectl --namespace="${NAMESPACE}" get secret \
+      pgo-root-cacert -o jsonpath='{.metadata.ownerReferences[*].name}')
+      echo "$CURRENT_OWNERS"
+      if [[ "$CURRENT_OWNERS" == *"owner1"* ]]; then
+          echo "first if"
+          exit 1
+      fi
+      if [[ "$CURRENT_OWNERS" != *"owner2"* ]]; then
+          echo "second if"
```

Comment:
What error does the test print without the echoes here? Do we need to put an echo in each `if` to get a clear understanding why it failed?

Reply:
Per above, I'm removing the echo output. These in particular probably should've been removed or updated for more clarity.